src/key-derivation/__tests__/bip39.test.ts

```diff
@@ -95,7 +89,7 @@ describe('bip39', () => {
     ];
 
     for (const vector of vectors) {
-      expect(mnemonicToSeed(vector.mnemonic, vector.password || undefined)).to.equal(vector.seed);
+      expect(Mnemonic.toSeed(vector.mnemonic, vector.password || undefined)).to.equal(vector.seed);
```

Comment:
fix vector.password comparison here. eslint `strict-boolean-expressions` is being triggered.

Reply:
Yes, will do